src/Go/effects/netscriptGoImplementation.ts

```diff
@@ -179,8 +179,19 @@ export async function getAIMove(boardState: BoardState): Promise<Play> {
     await sleep(400);
     const aiUpdatedBoard = makeMove(boardState, result.x, result.y, GoColor.white);
 
-    // Handle the AI breaking. This shouldn't ever happen.
     if (!aiUpdatedBoard) {
+      // If the previous move was made by the AI, log it, even if it didn't come from this thread.
```

Comment:
I don't understand what you mean by "this thread." JS is inherently single-threaded.